packages/design-system/src/styles/components/_Table.scss

```diff
@@ -55,37 +61,60 @@
     word-wrap: normal;
   }
 
-  th,
-  td {
+  tr,
+  caption {
+    display: block;
+  }
+
+  td,
+  th {
     border: 0;
+    display: grid;
+    grid-template-columns: 42% auto;
+    position: relative;
+    z-index: 100;
+
+    // stylelint-disable selector-no-qualifying-type
+    &[data-title]::before {
+      // stylelint-enable selector-no-qualifying-type
+      content: attr(data-title);
+      font-weight: bold;
+      max-width: 40%;
+      width: 40%;
+    }
+
+    > * {
+      grid-column: 2;
+    }
   }
 
   tr {
+    background-color: color('white');
     border: 1px solid $color-gray;
+    border: 1px solid;
+    display: block;
     margin: 0 0 0.5rem;
+    padding-bottom: 2px;
+    position: relative;
+
+    &::after {
+      background-color: $color-gray-lightest;
+      bottom: 0;
+      content: '';
+      display: block;
+      left: 0;
+      max-width: 38%;
```

Comment:
same here, any reason why its 38 instead of 40?

Reply:
I think this could be changed to 40%